Dart/testData/generate/EqualsAndHashCode2.txt

```diff
@@ -0,0 +1,15 @@
+class EqualsAndHashCode2 {
+
+  @override
+  bool operator ==(Object other) {
+    if (identical(this, other)) {
+      return true;
+    }
+    return other is EqualsAndHashCode2;
+  }
+
+  @override
+  int get hashCode {
+    return 0;
+  }<caret>
```

Comment:
`==()` and `hashCode` for classes without fields look useless. May be hide the action in classes without fields and do not allow OK in dialog without selection?